src/components/layout/Header/HeaderContact.module.scss

```diff
@@ -0,0 +1,31 @@
+@import '../../../assets/styles/variables';
```

Comment:
Pls research around scss-aliases.
We can simplify scss import to make it like `@import 'variables';` from any folder.